back/flag-service/test/util/registerAndLogin.ts

```diff
@@ -1,7 +1,7 @@
 import { INestApplication } from '@nestjs/common';
 import request from 'supertest';
 
-export async function registerAndLogin(app: INestApplication, email: string, password: string, nickname: string): Promise<string> {
+export async function registerAndLogin(app: INestApplication, email: string, password: string, nickname: string): Promise<{jwt: string, user: any}> {
```

Comment:
Should have spaces after the opening bracket and before the closing bracket of object literals.